src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,33 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { Todo } from "../../types/Todo";
+import { UserInfo } from "../UserInfo";
+import classNames from 'classnames';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+      {user && (
+        <UserInfo user={user} />
+      )}
+    </article>
```

Comment:
don't forget about code style
```suggestion
    <article
      className={classNames('TodoInfo', {
          'TodoInfo--completed': completed,
        })}
    >
      <h2 className="TodoInfo__title">
        {title}
      </h2>
      
      {user &&  <UserInfo user={user} />}
    </article>
```